packages/shared/src/components/modals/bookmark/BookmarkFolderModal.tsx

```diff
@@ -81,7 +81,7 @@ const PlusCTA = ({ folderCount }: { folderCount: number }) => {
 
 const ModalTitle = () => (
   <>
-    <ModalHeader.Title>New Folder</ModalHeader.Title>
+    <ModalHeader.Title className="typo-title3">New Folder</ModalHeader.Title>
```

Comment:
Oh this actually support kind.
I think it might be because the dot notation?

Shouldn't have to pass it technically.

Reply:
I've checked how the modal works and how is implemented. Issue is that we are not using the default title props because we need to pass also the `Plus` badge other than a string. 
I implemented a solution passing a ReactNode to modal instead so we can keep the same logic. 

cc: @AmarTrebinjac